src/test/java/lambda/part2/exercise/ArrowNotationExercise.java

```diff
@@ -23,26 +23,49 @@ public void getAge() {
     public void compareAges() {
         // TODO use BiPredicate
         // compareAges: (Person, Person) -> boolean
+        final BiPredicate<Person, Person> compareAges = (p1, p2) -> {
+            if (Integer.compare(p1.getAge(), p2.getAge()) == 0) {

```

Comment:
Use `a == b` instead of `Integer.compare(a, b) == 0`.
Use `return t` instead of
```
if (t) {
  return true;
} else {
  return false;
}
```